liquidator/liquidator.js

```diff
@@ -26,7 +26,7 @@ class Liquidator {
 
   // Queries underCollateralized positions and performs liquidations against any under collateralized positions.
   queryAndLiquidate = async priceFunction => {
-    const contractTime = await this.empContract.methods.getCurrentTime().call();
+    const contractTime = this.empClient.getLastUpdateTime();
```

Comment:
rather than asking the chain what the current time is use the time that the client was last updated to ensure consistency.

Reply:
this is a great feature. If `liquidator` is downstream client of `EMPClient` then makes sense to ask it for the time